cmd/limactl/copy.go

```diff
@@ -80,7 +82,13 @@ func copyAction(cmd *cobra.Command, args []string) error {
 			if inst.Status == store.StatusStopped {
 				return fmt.Errorf("instance %q is stopped, run `limactl start %s` to start the instance", instName, instName)
 			}
-			scpArgs = append(scpArgs, fmt.Sprintf("scp://%s@127.0.0.1:%d/%s", u.Username, inst.SSHLocalPort, path[1]))
+			// Only OpenSSH version 8.0 and later support using source and target as scp:// URIs
+			if !sshutil.DetectOpenSSHVersion().LessThan(*semver.New("8.0.0")) {
+				scpArgs = append(scpArgs, fmt.Sprintf("scp://%s@127.0.0.1:%d/%s", u.Username, inst.SSHLocalPort, path[1]))
+			} else {
+				scpArgs = append(scpArgs[:flagsEndIdx], append([]string{"-P", fmt.Sprintf("%d", inst.SSHLocalPort)}, scpArgs[flagsEndIdx:]...)...)
+				scpArgs = append(scpArgs, fmt.Sprintf("%s@127.0.0.1:%s", u.Username, path[1]))
+			}
```

Comment:
Can we just unconditionally use the legacy-compatible form, even when OpenSSH binary is >= 8.0?